package.json

```diff
@@ -7,6 +7,8 @@
     "test": "node test | tap-summary",
     "buildSample": "browserify example/index.js -o example/bundle.js",
     "ci": "node test | tap-summary --no-progress",
+    "clean:blocklist": "node src/clean-config.js blocklist > ._tmp_clean_config.json && mv ._tmp_clean_config.json src/config.json",
+    "clean:allowlist": "node src/clean-config.js allowlist > ._tmp_clean_config.json && mv ._tmp_clean_config.json src/config.json",
```

Comment:
(nitpick) passing the file path as an argument and using `fs.readFileSync and fs.writeFileSync` would make this cleaner.

Reply:
Agreed on skipping yargs. I realize you don't need to point to a path even, because it's assumed to be `./config.json`.
But writing back to a file is the default behavior and you could switch to terminal output based on an extra param then.